mkosi

```diff
@@ -1626,12 +1626,17 @@ SigLevel    = Required DatabaseOptional
     run_pacstrap(packages)
 
     if args.bootable:
-        # Patch mkinitcpio configuration so: 1) we remove autodetect and
+        # Patch mkinitcpio configuration so:
+        # 1) we remove autodetect and
         # 2) we add the modules needed for encrypt.
-        patch_file(os.path.join(workspace, "root", "etc/mkinitcpio.conf"),
-                   lambda line: "HOOKS=\"systemd modconf block sd-encrypt filesystems keyboard fsck\"\n" if line.startswith("HOOKS=") and args.encrypt == "all" else
-                                "HOOKS=\"systemd modconf block filesystems fsck\"\n"                     if line.startswith("HOOKS=") else
-                                line)
+        def jj(line: str) -> str:
```

Comment:
I used `jj` because the lambdas in `set_root_password()` were already named `jj`, so I just kept that name for those, and used it in the other places too, for consistency.  If I'd made up the name "from scratch", I'd probably have named it `patchfn`

It looks like `jj` was originally introduced by @keszybz, you'd have to ask him.